.circleci/config.yml

```diff
@@ -376,6 +385,41 @@ commands:
             ICONS=pro yarn build-storybook
             ls -lRa ./storybook-build
 
+  e2e_build:
+    parameters:
+      test_runner:
+        type: enum
+        enum:
+          - testcafe
+          - playwright
+    steps:
+      - restore_repo
+      - setup_remote_docker:
+          version: *remote_docker_version
+          docker_layer_caching: false
+      - run:
+          name: Build and push E2E << parameters.test_runner >> image
+          working_directory: *workspace_compose
+          command: |
+            ecr-login
+            # The image is not expected to chage too often so we use :latest for all cache
+            E2E_IMAGE="307238562370.dkr.ecr.eu-west-1.amazonaws.com/evaka/e2e-<< parameters.test_runner >>:latest"
```

Comment:
Suggestion: move the Docker registry part of image names to a pipeline parameter (like `workspace_root`).

Reply:
Unified the parameters for job and parametrized the registry.